docs/conf.py

```diff
@@ -41,7 +43,7 @@
 
 # General information about the project.
 project = u'Django Ratelimit'
-copyright = u'2014, James Socol'
+copyright = u'2014 - %s, James Socol' % (datetime.date.today().year)
```

Comment:
I'm not sure there's really any case for this. If someone downloads the source and renders the docs in 2099, it'll claim copyright up until 2099 even if the last dev work was 2018.

Reply:
OK dropping it.